vita3k/shader/src/translator/data.cpp

```diff
@@ -592,8 +592,6 @@ bool USSETranslatorVisitor::vldst(
     spv::Id source_1 = load(inst.opr.src1, 0b1, 0);
     spv::Id source_2 = load(inst.opr.src2, 0b1, 0);
 
-    source_1 = m_b.createBinOp(spv::OpIMul, m_b.makeIntType(32), source_1, m_b.makeIntConstant(4));
```

Comment:
This actually contradicts sunho's 4dee9931ac0cd0aa205516b0295bb73214f83869 T_T. So I want to see if any regressions are caused by this.

Reply:
IIRC, it was added to fix glitches in BlazBlue. We can check it out.

Reply:
Maybe we can just scale it by the factor of `get_data_type_size(type_to_ldst)` That way, we don't need `if (inst.opr.src1.bank == RegisterBank::IMMEDIATE) { ... }` and it would work for other banks as well.

Reply:
What do you mean? It definitely is not in data type unit for indexing by registers. I checked both BlazBlue and VirtuaTennis.

Reply:
`inst.opr.src1.num *= get_data_type_size(type_to_ldst);` isn't this for unit conversion of ldr/srt offset?

Reply:
ah I just read the code regarding index thing right below. Never mind about my comments.